blazor/smith-chart/smith-chart-marker.md

```diff
@@ -53,7 +53,7 @@ Using the [SmithChartSeriesMarker](https://help.syncfusion.com/cr/blazor/Syncfus
 * [Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarker.html#Syncfusion_Blazor_Charts_SmithChartSeriesMarker_Height) - Used to customize the height of the marker.
 * [Fill](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarker.html#Syncfusion_Blazor_Charts_SmithChartSeriesMarker_Fill) - Used to customize the fill color of the marker.
 * [Opacity](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarker.html#Syncfusion_Blazor_Charts_SmithChartSeriesMarker_Opacity) - Used to customize the opacity of the marker.
-* [SmithChartSeriesMarkerBorder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarkerBorder.html#properties) - Used to control the width and the color of the marker's border using the [Color](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarkerBorder.html#Syncfusion_Blazor_Charts_SmithChartSeriesMarkerBorder_Color) and the [Width](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SmithChartSeriesMarkerBorder.html#Syncfusion_Blazor_Charts_SmithChartSeriesMarkerBorder_Width) properties.
```

Comment:
check this link 
<img width="982" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/61148953/02a5378e-9f51-4ab4-a123-de223061d652">

Reply:
Have  reverted the changes